### PR TITLE
fix sha error in npm

### PR DESCRIPTION
**Pull request by TevinWang**
Currently getting a `npm ERR! code EINTEGRITY` on merge, need to fix asap.

https://stackoverflow.com/questions/75905223/github-action-npm-install-give-code-eintegrity-integrity-checksum-failed

### CODE REVIEW

File: .github/workflows/docs_test.yml [hunk 1]
```diff
@@ -76,8 +76,9 @@ jobs:
     - name: Install node dependencies
       run: |
         cd docs/test
-        npm cache clean --force
-        npm ci
+        npm cache clean -f
+        rm package-lock.json
```
Comment (eddyxu) [inline .github/workflows/docs_test.yml:80]: do we still have this file if the package-lock.json is deleted from git

File: docs/test/package.json [hunk 2]
```diff
@@ -4,12 +4,10 @@
...
   "license": "ISC",
-  "devDependencies": {
-    "vectordb": "https://gitpkg.now.sh/lancedb/lancedb/node?main"
-  },
   "dependencies": {
     "fs": "^0.0.1-security",
     "glob": "^10.2.7",
-    "path": "^0.12.7"
+    "path": "^0.12.7",
+    "vectordb": "https://gitpkg.now.sh/lancedb/lancedb/node?main"
```
Comment (eddyxu) [inline docs/test/package.json:11]: Curious why vectordb was dev dependency before

File: .github/workflows/docs_test.yml (hunk 1 above)
Comment (TevinWang) [reply .github/workflows/docs_test.yml]: Yeah we do... deleted!

File: docs/test/package.json (hunk 2 above)
Comment (TevinWang) [reply docs/test/package.json:11]: Mistake on my part, should have changed it to dev, was doing some experimenting

**Review by eddyxu — APPROVED** [review]

**Pull request merged**